tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -528,6 +543,13 @@ static void stream_disabled(struct bt_bap_stream *stream)
 	struct audio_stream *a_stream = CONTAINER_OF(stream, struct audio_stream, stream);
 
 	LOG_DBG("Disabled stream %p", stream);
+
+	if (stream->dir == BT_AUDIO_DIR_SINK) {
+		/* Stop send timer */
+		k_work_cancel_delayable(&a_stream->audio_clock_work);
+		k_work_cancel_delayable(&a_stream->audio_send_work);
+	}
```

Comment:
Is this used for both the unicast client and the unicast server, or just one of them?